pyproject.toml

```diff
@@ -36,7 +36,7 @@ classifiers = [
 ]
 dependencies = [
     "async_lru>=1.0.0",
-    "httpx>=0.25.0",
+    "httpx>=0.28.0",
```

Comment:
This was pretty bad breakage for users as extension manager stopped working on new installs.
Do we want to use `~=0.28.0` to avoid this in the future?

Reply:
We should have had a deprecation warning though.
If we add an upper bound, we will have to manually upgrade `httpx`, but I don't have a strong opinion.

Reply:
I think this will ease maintenance burden, and we can always unpin it later.